tests/test_nn.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+
+import torch
+from parameterized import parameterized
+
+from dwave.plugins.torch.nn import LinearBlock, SkipLinear, store_config
+from tests.utils import model_probably_good
+
+
+class TestUtils(unittest.TestCase):
+
+    def test_store_config(self):
+        # Simple case
+        class MyModel(torch.nn.Module):
+            @store_config
+            def __init__(self, a, b=1, *, x=4, y='hello'):
+                super().__init__()
+
+        model = MyModel(a=123, x=5)
+        self.assertDictEqual(dict(model.config),
+                             {"a": 123, "b": 1, "x": 5, "y": "hello", "module_name": "MyModel"})
+
+        # Nested case
+        class InnerModel(torch.nn.Module):
+            @store_config
+            def __init__(self, a, b=1, *, x=4, y='hello'):
+                super().__init__()
+
+        class OuterModel(torch.nn.Module):
+            @store_config
+            def __init__(self, module_1, module_2=None):
+                super().__init__()
+
+        module_1 = InnerModel(a=123, x=5)
+        module_2 = InnerModel(a="second", y="lol")
+        model = OuterModel(module_1, module_2)
+        self.assertDictEqual(dict(model.config),
```

Comment:
Does this work well comparing nested `MappingProxyType` objects? I wonder if you'd need to test e.g.,
```python
self.assertDictEqual(dict(model.config[module_1]), {"a": 123, "b": 1:, "x": 5, "y": "hello"}
self.assertDictEqual(dict(model.config[module_2]), {"a": "second", "b": 1:, "x": 4, "y": "lol"}
```
separately as well.